vendor/bat-native-ledger/src/bat/ledger/internal/contribution/phase_two.cc

```diff
@@ -245,7 +246,38 @@ void PhaseTwo::PrepareBatch(
       callback);
 }
 
+void PhaseTwo::PrepareBallots(
```

Comment:
is this intentionally that you named it the same as a function above?

Reply:
because they are doing different things

Reply:
No, that was unintentional. I'll rename this new function.